department-of-veterans-affairs/va.gov-cms
# VACMS-15308: update color variables in css (includes VACMS-14268).

## Description

Relates to #15560 and #14753 

## Testing done
Local test run to make sure theme and storybook compile correctly.

## Screenshots


## QA steps

What needs to be checked to prove this works?
Accessibility and design to check global items and spot check pages to ensure look is acceptable.
What needs to be checked to prove it didn't break any related things?
No behavior should have changed.

### Definition of Done


### Select Team for PR review



### Does this PR need review from a Product Owner
